sse2neon.h

```diff
@@ -1219,8 +1271,8 @@ FORCE_INLINE __m128i _mm_shuffle_epi8(__m128i a, __m128i b)
     __asm__ __volatile__(
         "vtbl.8  %e[ret], {%e[tbl], %f[tbl]}, %e[idx]\n"
         "vtbl.8  %f[ret], {%e[tbl], %f[tbl]}, %f[idx]\n"
-        : [ret] "=&w"(ret)
-        : [tbl] "w"(tbl), [idx] "w"(idx_masked));
+        : [ ret ] "=&w"(ret)
```

Comment:
clang-format-10 is known to a bit weird for inline assembly. You can use [clang-format-6](https://releases.llvm.org/6.0.1/tools/clang/docs/ClangFormat.html).